### PR TITLE
Feat/Add warning for low fee

### PR DESCRIPTION
**Pull request by JDonadio**
Fixes https://github.com/bitpay/copay/issues/5693

copay_-_copay_bitcoin_wallet

copay_-_copay_bitcoin_wallet

### CODE REVIEW

File: www/views/tx-details.html [hunk 1]
```diff
@@ -84,6 +84,10 @@
             <span ng-if="toggleFeeFiat">{{feeFiatStr}}</span>
           </span>
         </div>
+        <div class="item low-fees" ng-if="btx.action == 'received' && btx.lowFees">
```
Comment (gabrielbazan7) [inline www/views/tx-details.html]: also : `btx.confirmations === 0`

Comment (dabura667) [inline www/views/tx-details.html]: I agree `btx.confirmations === 0` should also be a condition.

Comment (gabrielbazan7) [inline www/views/tx-details.html]: Ok. It's done here https://github.com/bitpay/bitcore-wallet-service/pull/650

**Comment by isocolsky** [comment]
ACK

**Pull request merged**